umbral/keys.py

```diff
@@ -54,7 +55,8 @@ def from_bytes(cls, key_data: bytes, params: UmbralParameters=None,
         if params is None:
             params = default_params()
 
-        key_bytes = base64.urlsafe_b64decode(key_data)
+        if decoder:
```

Comment:
So - if the decoder is `None` we don't use one at all?  Do we want to provide a default?